pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[project]
+name = "rulesengine"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.11"
+dependencies = [
+    "certifi>=2025.10.5",
+    "django>=5.2.7",
+    "django-extensions>=4.1",
+    "djangoql>=0.18.1",
+    "idna>=3.11",
+    "jsonschema>=4.25.1",
+    "psycopg[binary]>=3.2.11",
```

Comment:
Now that we've got a proper dependency specification, I wonder if it's worth making psycopg an extra instead of mandatory - since the default is currently sqlite.